readme.md

```diff
@@ -26,6 +31,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
+You can use the $('selector').ajaxForm or .ajaxSubmit or .serialize with .post. First you need to use the selector for the form and pass it through the ajax post function to send it back to the database you are working with.
 
```

Comment:
This looks like the code using a JQuery plugin, what would the steps be for submission via AJAX?